qark/plugins/Manifest_File_Checks.py

```diff
@@ -1,22 +1,71 @@
-import sys, os, re
-import qarkMain
-
-sys.path.insert(0, os.path.dirname(os.path.realpath(__file__)) + '../lib')
-
 from yapsy.IPlugin import IPlugin
 from plugins import PluginUtil
 from modules import common
 from lib.pubsub import pub
+import sys
+import os
+import re
+import qarkMain
+
+sys.path.insert(0, os.path.dirname(os.path.realpath(__file__)) + '../lib')
 
 
-class ManifestFilePlugin(IPlugin):
+def ordered_broadcast(list_orderedBR):
```

Comment:
Remove these as methods and just have string variables named like the method name. For instance, this method would become 

```python
ordered_braodcast_issue = "Data Injection due to exported Broadcast Receiver.\n"
"Default priority of exported receiver is 0. Since, the higher priority receivers respond first"
" and forward it to lower priority receivers, a malicious receiver with high priority can intercept the "
"message change it and forward it to lower priority receivers.\n{}\n")
```

Then when you need to format stuff into it, just call `ordered_broadcast_issue.format(your_stuff)`

This avoids creating unnecessary functions that can be easily solved using variables for it instead.